### ISSUE TITLE
Image urls on homepage are hardcoded

### ISSUE THREAD

**Issue by advancedwpuser** [advancedwp/awp-2015#6]
Needs to use get_template_directory_uri()

**Comment by advancedwpuser** [comment]
Actually, this is related to #1 since the images are hard-coded via the Custom Fields.

**Comment by kevinwhoffman** [comment]
Yes, ACF stores image fields as IDs, so this issue would be resolved by switching to ACF. However, since Custom Fields Suite (the current custom fields plugin) stores images as URLs, we can't simply map existing custom fields to ACF fields as we originally planned. There is definitely going to be some data re-entry involved.

**Comment by advancedwpuser** [comment]
Not that much though. It's just for the homepage. That's why I want to get it done now before we build out more pages and make it even MORE complex.
